Assets/Content/Entities/Humanoids/Human/RigidBodyInspector.cs

```diff
@@ -6,7 +6,7 @@ namespace SS3D.Content.Creatures.Human
     // This adds a button in the editor to test ragdolling
     #if UNITY_EDITOR
     [CustomEditor(typeof(HumanRagdoll))]
-    public class RigidBodyInspector : Editor
+    public class RigidBodyInspector : UnityEditor.Editor
```

Comment:
what's this change for ?

Reply:
After adding the assemblies, this file could not decide if it needs UnityEditor.Editor or UnityEngine.Editor for some reason, as it imports both. So I just specified it to get rid of the ambiguity error.

Reply:
great